src/query/service/src/servers/mysql/mysql_interactive_worker.rs

```diff
@@ -420,7 +421,15 @@ impl<W: AsyncWrite + Send + Unpin> InteractiveWorkerBase<W> {
         let do_query = self.do_query(&init_query).await;
         match do_query {
             Ok((_, _)) => Ok(()),
-            Err(error_code) => Err(error_code),
+            Err(error_code) => {
+                if error_code.code() == ErrorCode::UNKNOWN_DATABASE {
```

Comment:
This can resolve :

```
create database Tt;
use Tt; -- error: database Tt not exists.
```

But it also has bad case:

```
MySQL [(none)]> create database Rustlang;
Query OK, 0 rows affected (0.026 sec)

MySQL [(none)]> create database `Rustlang`;
Query OK, 0 rows affected (0.036 sec)

MySQL [(none)]> create table Rustlang.t_low(id int);
Query OK, 0 rows affected (0.044 sec)

MySQL [(none)]> create table `Rustlang`.t_up(id int);
Query OK, 0 rows affected (0.034 sec)

MySQL [(none)]> use Rustlang;
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
MySQL [Rustlang]> show tables; --- use db without quota, in mind the result of show tables should be: t_low
+--------------------+
| tables_in_rustlang |
+--------------------+
| t_up               |
+--------------------+
1 row in set (0.038 sec)

```

Reply:
> This can resolve :
> 
> ```
> create database Tt;
> use Tt; -- error: database Tt not exists.
> ```
> 
> But it also has bad case:
> 
> ```
> MySQL [(none)]> create database Rustlang;
> Query OK, 0 rows affected (0.026 sec)
> 
> MySQL [(none)]> create database `Rustlang`;
> Query OK, 0 rows affected (0.036 sec)
> 
> MySQL [(none)]> create table Rustlang.t_low(id int);
> Query OK, 0 rows affected (0.044 sec)
> 
> MySQL [(none)]> create table `Rustlang`.t_up(id int);
> Query OK, 0 rows affected (0.034 sec)
> 
> MySQL [(none)]> use Rustlang;
> Reading table information for completion of table and column names
> You can turn off this feature to get a quicker startup with -A
> 
> Database changed
> MySQL [Rustlang]> show tables; --- use db without quota, in mind the result of show tables should be: t_low
> +--------------------+
> | tables_in_rustlang |
> +--------------------+
> | t_up               |
> +--------------------+
> 1 row in set (0.038 sec)
> ```

BTW:

This may be confusing for users:

```
MySQL [Rustlang]> use Rustlang;
Database changed
MySQL [Rustlang]> show tables;
+--------------------+
| tables_in_rustlang |
+--------------------+
| t_up               |
+--------------------+
1 row in set (0.036 sec)

MySQL [Rustlang]> show tables from Rustlang;
+--------------------+
| tables_in_rustlang |
+--------------------+
| t_low              |
+--------------------+
1 row in set (0.043 sec)

```

Reply:
cc @leiysky @BohuTANG PTAL

Reply:
I think in the main branch this is a bug since the `database` identifier should be normalized. This fix will resolve the issue.